substrate/frame/support/test/tests/pallet.rs

```diff
@@ -742,10 +742,40 @@ impl pallet5::Config for Runtime {
 	type RuntimeOrigin = RuntimeOrigin;
 }
 
+#[derive(Clone, Debug, codec::Encode, codec::Decode, PartialEq, Eq, scale_info::TypeInfo)]
+pub struct AccountU64(u64);
```

Comment:
more evidence that this should def. be moved to `sp_runtime::testing` :)

Reply:
I agree.

Reply:
Moved under `frame_support::crypto::mock::AccountU64` because of some interdependencies, but it's now under a common path.